src/dioptra/restapi/v1/plugins/service.py

```diff
@@ -466,15 +420,33 @@ def delete(self, plugin_id: int, **kwargs) -> dict[str, Any]:
         if plugin_resource is None:
             raise EntityDoesNotExistError(PLUGIN_RESOURCE_TYPE, plugin_id=plugin_id)
 
+        from dioptra.restapi.v1.plugins.service import PluginIdFileService
+
+        plugin_id_file_service = PluginIdFileService(
```

Comment:
If we need `PluginIdFileService` it should be injected as we do the other services.